lib/chef/provisioning/aws_driver/version.rb

```diff
@@ -1,7 +1,7 @@
 class Chef
 module Provisioning
 module AWSDriver
-  VERSION = '1.3.0.dev.0'
+  VERSION = '1.2.1'
```

Comment:
Yeah - I plan to quickly release 1.2.1 after merging this to master, and then change it back to `1.2.0.dev.1` post-release.